README.md

```diff
@@ -2,9 +2,9 @@
 
 # NuPIC C++ Core Library
 
-[![Linux/OSX Build Status](https://travis-ci.org/htm-community/nupic.cpp.svg?branch=master)](https://travis-ci.org/htm-community/nupic.cpp)
-[![OSX CircleCI](https://circleci.com/gh/htm-community/nupic.cpp/tree/master.svg?style=svg)](https://circleci.com/gh/htm-community/nupic.cpp/tree/master)
-[![Windows Build status](https://ci.appveyor.com/api/projects/status/59f87and1x0ugss9/branch/master?svg=true)](https://ci.appveyor.com/project/htm-community/nupic-cpp/branch/master)
+[![Travis: Linux Build Status](https://travis-ci.org/htm-community/nupic.cpp.svg?branch=master)](https://travis-ci.org/htm-community/nupic.cpp)
+[![CircleCI: OSX, ARM64 Build Status](https://circleci.com/gh/htm-community/nupic.cpp/tree/master.svg?style=svg)](https://circleci.com/gh/htm-community/nupic.cpp/tree/master)
```

Comment:
Sorry, I don't think this change is correct, as
- OSX is one of our main supported platforms
- and we cannot(?) distinguish build status of the jobs in CircleCI
This changes makes the two presented together, which is not needed/correct (currently ARM failing, while OSX is ok) 

https://shields.io/category/build 
might have a way, but I didn't find it.